manifests/database/postgresql.pp

```diff
@@ -20,7 +20,7 @@
     class { '::postgresql::server':
       ip_mask_allow_all_users => '0.0.0.0/0',
       listen_addresses        => $listen_addresses,
-      port                    => $database_port,
+      port                    => 0 + $database_port,
```

Comment:
We're going to roll a major release soon and bring the minimum required version to Puppet 4, so `scanf` should be ok here.